src/sentry/static/sentry/app/views/monitors/monitors.tsx

```diff
@@ -78,15 +80,14 @@ class Monitors extends AsyncView {
             </NewMonitorButton>
           </HeaderTitle>
           <StyledSearchBar
-            organization={organization}
-            query={(location.query && location.query.query) || ''}
+            query={(qs.parse(location.search)?.query as string) ?? ''}
```

Comment:
Are you not able to use `decodeScalar(location.query.query) ?? ''` here?

Reply:
o I did not know about this one 👀